<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dodaj_Klienta</class>
 <widget class="QDialog" name="Dodaj_Klienta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dodaj klienta</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ikonki/16x16/klienci.png</normaloff>ikonki/16x16/klienci.png</iconset>
  </property>
  <widget class="QLineEdit" name="Imie_lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>371</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Nazwisko_lineedit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>371</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Nazwa_Firmy_lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>371</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Adres_lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>371</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="NIP_lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>371</width>
     <height>27</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Regon_lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>371</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Nr_Konta_lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>371</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Imie_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>73</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>ImiÄ™</string>
   </property>
  </widget>
  <widget class="QLabel" name="Nazwisko_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwisko</string>
   </property>
  </widget>
  <widget class="QLabel" name="Nazwa_Firmy_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa firmy</string>
   </property>
  </widget>
  <widget class="QLabel" name="Adres_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Adres</string>
   </property>
  </widget>
  <widget class="QLabel" name="NIP_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>73</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>NIP</string>
   </property>
  </widget>
  <widget class="QLabel" name="Regon_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>73</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Regon</string>
   </property>
  </widget>
  <widget class="QLabel" name="Nr_Konta_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Nr konta</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>189</width>
     <height>34</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="telefon_kontaktowy_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefon kontaktowy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="telefon_kontaktowy_lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>341</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Imie_lineEdit</tabstop>
  <tabstop>Nazwisko_lineedit</tabstop>
  <tabstop>Nazwa_Firmy_lineEdit</tabstop>
  <tabstop>Adres_lineEdit</tabstop>
  <tabstop>NIP_lineEdit</tabstop>
  <tabstop>Regon_lineEdit</tabstop>
  <tabstop>Nr_Konta_lineEdit</tabstop>
  <tabstop>telefon_kontaktowy_lineEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
